### DISCUSSION TITLE
No code editor?

### DISCUSSION THREAD

**Discussion by ki4oir** [community/community#8309] [Mobile] [answered]
Why doesn't the iOS app allow me to create a new branch or edit files? This should be part of the features.

**Comment by fishooo0815** [comment]
I want to Edith Files with the Mobile APP.

**Comment by eliperkins** [comment, marked as answer]
Thanks for the feedback @ki4oir! Editing code is definitely on our radar, and we want to make sure that we do it right! Creating branches seems like a great idea to pair with editing code.

I've added your notes to our internal tracking issue about editing code within the app. I'll talk to the team about this some more!

**Reply by ki4oir** [reply]
I'll be waiting. Thanks.

**Reply by MarciaRieferJohnston** [reply]
I'll be waiting too. I was stunned today when I went to edit a readme file and discovered that I couldn't. I thought I was doing something wrong.

**Comment by spaltrowitz** [comment]
this has now shipped! https://github.blog/changelog/2022-11-15-github-for-ios-edit-files-from-browse-code cc @ki4oir @MarciaRieferJohnston @fishooo0815
